_includes/docs/user-guide/install/ce-config-tables.md

```diff
@@ -0,0 +1,1274 @@
+
+
+####  Server common properties
+<table>
```

Comment:
please add extra space. Also add some spaces to make those files a bit more human readable. for example:

```
<table>
  <thead>
      <tr>
          <td style="width: 25%"><b>Parameter</b></td><td style="width: 30%"><b>Environment Variable</b></td><td style="width: 15%"><b>Default Value</b></td><td style="width: 30%"><b>Description</b></td>
      </tr>
  </thead>
  <tbody>
      <tr>
          <td>server.address</td>
          <td>HTTP_BIND_ADDRESS</td>
          <td>0.0.0.0</td>
          <td>HTTP Server bind address</td>
      </tr>
```